presto-hive-hadoop2/bin/run_hive_s3_tests.sh

```diff
@@ -8,6 +8,7 @@ cleanup_docker_containers
 start_docker_containers
 
 # insert AWS credentials
+# TODO replace core-site.xml.s3-template with apply-overrides
```

Comment:
What is `apply-overrides`?

Reply:
will update to `apply-site-xml-override.sh`